src/v2i-hub/ODEForwardPlugin/CMakeLists.txt

```diff
@@ -5,3 +5,18 @@ SET (TMX_PLUGIN_NAME "ODEForwardPlugin")
 BuildTmxPlugin ( )
 
 TARGET_LINK_LIBRARIES (${PROJECT_NAME} tmxutils rdkafka++)
+
+#############
+## Testing ##
+#############
+enable_testing()
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
NIT: instead of using global commands in CMake that modify path for all targets, it is normally better to use target based commands like target_include_directories .